video_creation/background.py

```diff
@@ -3,28 +3,38 @@
 import re
 from pathlib import Path
 from random import randrange
-from typing import Any, Tuple
+from typing import Any, Tuple,Dict
 
-from moviepy.editor import VideoFileClip
+from moviepy.editor import VideoFileClip,AudioFileClip
 from moviepy.video.io.ffmpeg_tools import ffmpeg_extract_subclip
 from utils import settings
 from utils.console import print_step, print_substep
 import yt_dlp
 
-# Load background videos
-with open("./utils/backgrounds.json") as json_file:
-    background_options = json.load(json_file)
+def load_background_options():
+    background_options = {}
+    # Load background videos
+    with open("./utils/background_videos.json") as json_file:
+        background_options["video"] = json.load(json_file)
 
-# Remove "__comment" from backgrounds
-background_options.pop("__comment", None)
+    # Load background audios
+    with open("./utils/background_audios.json") as json_file:
+        background_options["audio"] = json.load(json_file)
+    
+    # Remove "__comment" from backgrounds
+    background_options["video"].pop("__comment", None)
```

Comment:
just use del, no need for pop or assignments.

Reply:
Did:
```
    # Remove "__comment" from backgrounds
    del background_options["video"]["__comment"]
    del background_options["audio"]["__comment"]
```